.travis.yml

```diff
@@ -15,7 +15,8 @@ before_install:
     - wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
     - bash Miniconda3-latest-Linux-x86_64.sh -b
     - export PATH=/home/travis/miniconda3/bin:$PATH
-    - pip install --upgrade pip && pip install numpy scipy matplotlib yt mpi4py
+    - pip install --upgrade pip && pip install numpy scipy matplotlib mpi4py cython
+    - pip install git+https://github.com/yt-project/yt.git
```

Comment:
This was required because of a small change in the format of the plotfiles, when using 2 MPI procs.